### ISSUE TITLE
[Feature Request] Set expiration date for several entries

### ISSUE THREAD

**Issue by GambaJo** [nextcloud/tasks#875]
It would be great if I can mark several entries in the list an set the expiration date for all of the marked entries.
For example when there are some entries left from yesterday I want to set them for today.

**Comment by raimund-schluessler** [comment]
Do I correctly understand that you want to edit multiple tasks at once (batch editing)?

**Comment by GambaJo** [comment]
> 
> 
> Do I correctly understand that you want to edit multiple tasks at once (batch editing)?

Yes. I know this from wunderlist, where I can mark several entries and set the expiration date for all of them at once or put them all in another list.

**Comment by raimund-schluessler** [comment]
Then let's rename this issue to make that clear (since it's not only about the expiration date).

**Comment by c64bob** [comment]
This would be a great timesaver - just having the ability to add today's due date to all overdue tasks would be enough for me.

Thanks for all your great work!

**Comment by raimund-schluessler** [comment]
> just having the ability to add today's due date to all overdue tasks would be enough for me.

Out of curiosity and a bit off-topic: what’s the benefit of having the todays date instead of a date in the past? The tasks will still be shown as due today if they are overdue.

**Comment by c64bob** [comment]
> Out of curiosity and a bit off-topic: what’s the benefit of having the todays date instead of a date in the past? The tasks will still be shown as due today if they are overdue.
It's definitely a bit ocd, but I don't like the distinction this creates between overdue tasks and tasks that are due today...

**Comment by RokeJulianLockhart** [comment]
This would provide a more intuitive mechanism to accomplish https://github.com/nextcloud/tasks/pull/414#issuecomment-494138745.
